src/game/scenes/melee.c

```diff
@@ -403,15 +403,17 @@ void handle_action(scene *scene, int player, int action) {
                     // TODO in netplay, use the lobby names
                     strncpy_or_truncate(player1->pilot->name, lang_get(player1->pilot->pilot_id + 20),
                                         sizeof(player1->pilot->name));
+                    // TODO: lang: remove (the need for) newline stripping
                     // 1player name strings end in a newline...
                     if(player1->pilot->name[strlen(player1->pilot->name) - 1] == '\n') {
                         player1->pilot->name[strlen(player1->pilot->name) - 1] = 0;
                     }
                     strncpy_or_truncate(player2->pilot->name, lang_get(player2->pilot->pilot_id + 20),
                                         sizeof(player1->pilot->name));
```

Comment:
This line here still using player1 for the sizeof, although they would probably be the same size, it is confusing.

Reply:
Fixed